ArduCopter/baro_ground_effect.cpp

```diff
@@ -16,9 +16,9 @@ void Copter::update_ground_effect_detector(void)
     float des_speed_ne_ms = 0.0f;
     float des_climb_rate_ms = pos_control->get_vel_desired_U_ms();
 
-    if (pos_control->is_active_NE()) {
-        Vector3f vel_target_neu_ms = pos_control->get_vel_target_NEU_ms();
-        des_speed_ne_ms = vel_target_neu_ms.xy().length();
+    if (pos_control->NE_is_active()) {
+        Vector3f vel_target_ned_ms = pos_control->get_vel_target_NED_ms();
```

Comment:
```suggestion
        const Vector3f &vel_target_ned_ms = pos_control->get_vel_target_NED_ms();
```

(or just omit the variable?)